lib-multisrc/zerotheme/src/eu/kanade/tachiyomi/multisrc/zerotheme/ZeroTheme.kt

```
@@ -61,23 +75,49 @@ abstract class ZeroTheme(
 
     // =========================== Details =================================
 
+    override fun getMangaUrl(manga: SManga) = "$baseUrl/$mangaSubString/${manga.url.substringAfterLast("/")}"
```

Note: This method is called in the UI thread, so it might silently fail if it sends web request through `mangaSubString`.